<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdministratorAddWindow</class>
 <widget class="QMainWindow" name="AdministratorAddWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="AddButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>310</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="InumEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="OnumEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="MoneyEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>250</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="TimeLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>时间</string>
    </property>
   </widget>
   <widget class="QLabel" name="InumLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>入账账户</string>
    </property>
   </widget>
   <widget class="QLabel" name="OnumLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>出账账户</string>
    </property>
   </widget>
   <widget class="QLabel" name="MoneyLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>金额</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
